be/src/vec/common/hash_table/hash_table.h

```diff
@@ -277,13 +277,14 @@ struct HashTableGrower {
                                    ? fill_capacity
                                    : fill_capacity + 1);
 
-        size_degree = num_elems <= 1 ? initial_size_degree
-                                     : (initial_size_degree > fill_capacity ? initial_size_degree
-                                                                            : fill_capacity);
+        size_degree = (doris::vectorized::UInt8)(
```

Comment:
why change size_degree to uint32 or uint64?

Reply:
because size_degree = log2(xxx) when set_buf_size, so uint8 is enough.